docs/user-guide.rst

```diff
@@ -269,8 +273,30 @@ dictionary in the ``fields`` argument provided to
 JSON
 ~~~~
 
-You can send a JSON request by specifying the encoded data as the ``body``
-argument and setting the ``Content-Type`` header when calling
+You can send a JSON request by specifying the data as ``json`` argument,
```

Comment:
A bunch of the code examples in the previous sections use JSON and handle it like

```py
    print(json.loads(resp.data.decode("utf-8"))["headers"])
```

These can be simplified with the new `json()` method. Perhaps this section should be moved up and then the following examples could use it.